### PR TITLE
fix speaker details should be hidden in mobile #5962

### LINKED ISSUE #5869: [BUG] Speaker details on mobile without avatar
### Description

The user avatar does not display on the event, but the speaker details do.  If the avatar cannot fit on mobile, the speaker details should be hidden on mobile.

mobile-event-speaker-details

  

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Shaikh-Danish**
## Fixes Issue

Edit: closes #5869 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5967"></a>

### CODE REVIEW

**Comment by eddiejaoude** [comment]
I think it is for this issue https://github.com/EddieHubCommunity/LinkFree/issues/5869, I have updated the description (@Shaikh-Danish please link to the issue in the future to help the maintainers)

**Comment by Danish-Shaikhh** [comment]
Sure, sorry for the confusion.

File: components/event/EventCard.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default function EventCard({ event, username }) {
                   {event.name}
                 </Link>
                 {event.userStatus && (
-                  <div className="text-primary-low-medium italic">
+                  <div className="text-primary-low-medium italic group hidden lg:block">
```
Comment (eddiejaoude) [inline components/event/EventCard.js:51]: I don't think `group` is needed here?

Comment (eddiejaoude) [reply components/event/EventCard.js:51]:
```suggestion
                  <div className="text-primary-low-medium italic hidden lg:block">
```

**Pull request merged**